sql/core/src/test/resources/sql-tests/inputs/datetime.sql

```diff
@@ -149,7 +149,7 @@ select to_timestamp('2019-10-06 A', 'yyyy-MM-dd GGGGG');
 select to_timestamp('22 05 2020 Friday', 'dd MM yyyy EEEEEE');
 select to_timestamp('22 05 2020 Friday', 'dd MM yyyy EEEEE');
 select unix_timestamp('22 05 2020 Friday', 'dd MM yyyy EEEEE');
-select from_json('{"time":"26/October/2015"}', 'time Timestamp', map('timestampFormat', 'dd/MMMMM/yyyy'));
+select from_json('{"timestamp":"26/October/2015"}', 'timestamp Timestamp', map('timestampFormat', 'dd/MMMMM/yyyy'));
```

Comment:
why do we change this test?

Reply:
After this PR, dynamically set "spark.sql.ansi.enabled" actually takes effect in parsing phase. This query will failed parsing cause `time` is a reserved key word of SQL standard.

Reply:
`timestamp` is also reserved in ANSI. How about `ts`?

Reply:
done